aries-site/src/data/structures/Structure.js

```diff
@@ -0,0 +1,27 @@
+/* Utility functions to support page manipulations */
+
+export class Structure extends Array {
```

Comment:
File structure wise, would it make more sense for these functions to live in utils? I could go either way here, because the proximity is nice, but since we're calling them "utility functions" my first thought is that it'd live in utils.

Reply:
I modified the description a bit. If it's okay, I'd like to leave them as is for the time being since they are particularly related to the stucture file. Can migrate later if we see additional use cases.

Reply:
Sounds good to me.